en/companion_computer/auterion_skynode.md

```diff
@@ -0,0 +1,82 @@
+# Auterion Skynode
+
+[Skynode](https://auterion.com/product/skynode/) is a powerful flight computer that combines a mission computer, flight controller, video streaming, networking, and cellular connection, in a single tightly integrated device.
+
+![Auterion Skynode (Enterprise)](../../assets/companion_computer/auterion_skynode/skynode_enterprise_tdw.png)
+
+The onboard software is Auterion OS, consisting of an enterprise-hardened version of PX4 running on the flight controller, and an operating system with advanced management software running on the mission computer.
+The OS is managed by Auterion in production, with customer applications running as "add ons" in a safe sandbox within the mission computer.
+
+Auterion OS and Skynode allow seamless integration with Auterion's other software and fleet management products.
+
+For information about Auterion and Skynode:
+
+- [auterion.com](https://auterion.com/)
+- [Skynode](https://auterion.com/product/skynode/) (auterion.com)
+- Skynode Guides:
+  - [Manufacturer's Guide](https://docs.auterion.com/manufacturers/getting-started/readme) 
+  - [App Developer's Guide](https://docs.auterion.com/developers/getting-started/readme)
+
+
+## Skynode with Vanilla PX4
+
+Skynode comes with an Auterion-managed release of PX4.
+If you would like to try a more recent PX4 flight kernel, you can install the upstream "vanilla" PX4 from [PX4/PX4-Autopilot](https://github.com/PX4/PX4-Autopilot).
+
+Upstream PX4 will generally work, with the following caveats:
+
+- Configuration for your exact product may not be present.
+  You may lose the configuration for ESCs, batteries, sensor configuration, and so on.
+- Some parameters might have different default values in the PX4 release shipped with Auterion OS.
+- Features accessed by vendor-specific customisations running on the companion computer may not be present in PX4.
```

Comment:
Added to the end "for discussion":

```suggestion
- Features accessed by vendor-specific customisations running on the companion computer may not be present in PX4.
- Auterion only provides support for Skynode running its own Auterion-managed version of PX4.
```

Reply:
@MaEtUgR I think this covers the "not officially supported" even more - it makes clear that Auterion is the manufacturer and the circumstances under which they provide support.